lib/backend/memory/memory.go

```diff
@@ -261,6 +264,7 @@ func (m *Memory) PutRange(ctx context.Context, items []backend.Item) error {
 		}
 		if !m.Mirror {
 			event.Item.ID = m.generateID()
+			event.Item.Revision = backend.CreateRevision()
```

Comment:
Do we want a single revision for the whole `PutRange`? Should we just get rid of `PutRange` altogether? It's only used in the `PutRange` test of the backend suite.